bundle/regal/rules/style/detached_metadata_test.rego

```diff
@@ -0,0 +1,39 @@
+package regal.rules.style["detached-metadata_test"]
+
+import future.keywords.if
+
+import data.regal.ast
+import data.regal.config
+import data.regal.rules.style["detached-metadata"] as rule
+
+test_fail_detached_metadata if {
+	r := rule.report with input as regal.parse_module("p.rego", `
+package p
+
+# METADATA
+# description: allow is always true
+
+
```

Comment:
One empty line is bad, too? Just wondering because we're only testing no empty line vs two empty lines

Reply:
It's bad too! Added another test for that. Thanks 👍

Reply:
Was that above there before? Anyways, thanks!